build/schema.graphql

```diff
@@ -31,3 +31,17 @@ type Unpause @entity(immutable: true) {
   blockTimestamp: BigInt!
   transactionHash: Bytes!
 }
+
+type Request @entity(immutable: true) {
+  id: Bytes!
+  sender: Bytes!
+  requestId: BigInt!
+  ipfsHash: String!
+}
+
+type Delivery @entity(immutable: true) {
```

Comment:
I think the official name is "Deliver"

Reply:
This is schema, doesn't have to match the official name. And since this stored as an event that has already happened, I thought naming it delivery would make more sense

Reply:
Addressed [236d6cf](https://github.com/valory-xyz/ai-registry-mech-graph/pull/2/commits/236d6cf6f26c1262ea233bb56986acd37732ee32)